languages/common-lisp/concepts/arithmetic/about.md

```diff
@@ -1 +1,58 @@
-TODO: add information on arithmetic concept
+Common Lisp uses the standard arithmetic operators for most operations but is somewhat unique in using a "prefix-notation" as opposed to the more familiar "infix-notion". More visually:
+
+```lisp
+;; Infix-notation (non-lisp languages)
+1 + 2 + 3 + 4 + 5 ; => 15
+;; Prefix-notation (lisp languages)
+(+ 1 2 3 4 5) ; => 15
+```
+
+While prefix notion turns some operations like `2 + 2` into the somewhat unfamiliar `(+ 2 2)` form, it makes it much easier to operate on more than one number at a time.
+
+### Arithmetic Operators With a Single Argument
+
+As a small quirk, the `-` and `/` operators have a special meanings when applied to only one number:
+
+```lisp
+;; A single number passed to `-` is simply negated
+(- 4)   ; => -4
+(- -32) ; => 32
+;; A single number passed to `/` returns the reciprocal
+(/ 8)   ; => 1/8
+(/ 0.1) ; => 10.0
+```
+
+### Comparing Numbers (FIXME: Do I really belong here?)
```

Comment:
Would love to know if you think this is an alright place for this!

Reply:
I think it may be. With `sameness` we left out all the type specific equalitiy operators - assuming they'd be part of the relevant type concepts... Unless we create a new concept for numeric equality?